File: backend/src/main/java/com/urmyfood/backend/domain/model/ChatSession.java
package com.urmyfood.backend.domain.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class ChatSession extends BaseDomainModel {
    private Long shopId;
    private Long userId;
}

File: frontend/shared/src/main/java/com/urmyfood/shared/domain/model/ChatSession.kt
package com.urmyfood.shared.domain.model

data class ChatSession(
    val id: Long,
    val shopId: Long,
    val shopName: String,
    val shopAvatarUrl: String?,
    val userId: Long,
    val customerName: String,
    val customerAvatarUrl: String?,
    val lastMessage: String?,
    val lastMessageAt: String?,
    val unreadCount: Int
)
